### PR TITLE
DOMA-5516 create schema BankIntegrationOrganizationContext

### PR DESCRIPTION
**Pull request by VKislov**
`BankIntegrationOrganizationContext` gives access to all service users of a particular `integration`. Reflects the fact that the `integration` is connected to the `organization`
Screenshot 2023-03-10 at 15 13 56

This model is created by a real user when they select the `integration` on the home report page.

Screenshot 2023-03-10 at 15 19 02

After that, you can request the service user to connect any banking model to this `organization`.

### CODE REVIEW

**Review by AntonAL — APPROVED** [review]

File: apps/condo/domains/banking/access/BankIntegrationOrganizationContext.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+async function canReadBankIntegrationOrganizationContexts (args) {
+    const { authentication: { item: user }, originalInput, context } = args
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return true
+
+    if (user.type === SERVICE) {
+        // const bankIntegration = get(originalInput, ['organization', 'connect', 'id'])
+        // TODO(VKislov): need to check access for service user with checkBankIntegrationsAccessRights(context, user.id, [bankIntegration])
```
Comment (pahaz) [inline apps/condo/domains/banking/access/BankIntegrationOrganizationContext.js:23]: this pr?

File: apps/condo/domains/banking/access/BankIntegrationOrganizationContext.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+async function canManageBankIntegrationOrganizationContexts (args) {
+    const { authentication: { item: user }, originalInput, operation, itemId, context } = args
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return true
+
+    if (user.type === SERVICE) {
+        const bankIntegration = get(originalInput, ['organization', 'connect', 'id'])
+        // TODO(VKislov): need to check access for service user with checkBankIntegrationsAccessRights(context, user.id, [bankIntegration])
```
Comment (pahaz) [inline apps/condo/domains/banking/access/BankIntegrationOrganizationContext.js:42]: use `return false` if it's not a part of this pr

File: apps/condo/domains/banking/utils/clientSchema/BankIntegrationOrganizationContext.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    BankIntegrationOrganizationContextCreateInput,
+    BankIntegrationOrganizationContextUpdateInput,
+    QueryAllBankIntegrationOrganizationContextsArgs,
+} from '@app/condo/schema'
+
+import { generateReactHooks } from '@open-condo/codegen/generate.hooks'
+
+import { BankIntegrationOrganizationContext as BankIntegrationOrganizationContextGQL } from '@condo/domains/banking/gql'
+
+// TODO(codegen): write utils like convertToFormState and formValuesProcessor if needed, otherwise delete this TODO
```
Comment (pahaz) [inline apps/condo/domains/banking/utils/clientSchema/BankIntegrationOrganizationContext.ts:16]: !

File: apps/condo/domains/banking/access/BankIntegrationOrganizationContext.js (hunk 1 above)
Comment (VKislov) [reply apps/condo/domains/banking/access/BankIntegrationOrganizationContext.js:23]: No sir. Need code here from https://github.com/open-condo-software/condo/pull/2943. Added TODO not to forget

**Pull request merged**